config/samples/example-test.sh

```diff
@@ -234,8 +235,31 @@ function getPodImage() {
 	done
 }
 
+function getHelmRevision() {
+	namespace=$1
+	releaseName=$2
+	wantRevision=$3
+	START_TIME=$(date +%s)
```

Comment:
**Suggestion:** Avoid using uppercase variable names in shell scripts to prevent conflicts with environment variables. Use snake_case for variable names.
```suggestion
 start_time=$(date +%s)
```